### PR TITLE
Fix typos and format template code

### PR DESCRIPTION
**Pull request by couscousdude**
Hi, I really enjoy using this template!

I noticed a few minor spelling and grammatical mistakes in the template, which I fixed. 

I also ran [typstfmt](https://github.com/astrale-sharp/typstfmt) on the template file so that the source code is more readable and traversable in an editor where line-wrapping has not been enabled (eg. Vim).

### CODE REVIEW

File: template/main.typ [hunk 1]
```diff
@@ -53,12 +53,12 @@ These functions will overwrite the standard #link("https://typst.app/docs/refere
...
 
-They both look similar, the only difference is that 'Ilm uses more spacing between individual characters.
+They both look similar, the only difference being that 'Ilm uses more spacing between individual characters.
 
-If you prefer Typst's default spacing then you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
+If you prefer Typst's default spacing, you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
 
 == Footer
-If a page does not begin with a chapter then we display the chapter's name, to which the current section belongs to, in the footer. #link(<demo>)[Click here] to go to @demo down below and see the footer in action.
+If a page does not begin with a chapter, then we display the chapter's name, to which the current section belongs, in the footer. #link(<demo>)[Click here] to go to @demo down below and see the footer in action.
```
Comment (talal) [inline template/main.typ]: Here I am using the comma with a relative clause. See the section for relative clause on https://www.grammarly.com/blog/comma/ for reference.

Kindly revert this change.

File: template/main.typ [hunk 2]
```diff
@@ -103,7 +103,7 @@ For comparison, this is how the same table would look with Typst's default styli
 
 = Code
 == Custom font
-'Ilm uses the #link("https://typeof.net/Iosevka/")[_Iosevka_] font for raw text instead of the default _Fira Mono_. If Iosevka is not installed then the template will fall back to Fira Mono.
+'Ilm uses the #link("https://typeof.net/Iosevka/")[_Iosevka_] font for raw text instead of the default _Fira Mono_. If Iosevka is not installed, the template will fall back to Fira Mono.
```
Comment (talal) [inline template/main.typ]: As I stated previously, this is subjective, not a technical error.

Kindly revert this change.

File: template/main.typ [hunk 3]
```diff
@@ -53,12 +53,12 @@ These functions will overwrite the standard #link("https://typst.app/docs/refere
 Here is how Typst's own #std-smallcaps[smallcaps] and #std-upper[upper] look compared to the 'Ilm's variants:\
 #hide[Here is how Typst's own ] #smallcaps[smallcaps] and #upper[upper]
 
-They both look similar, the only difference is that 'Ilm uses more spacing between individual characters.
+They both look similar, the only difference being that 'Ilm uses more spacing between individual characters.
 
-If you prefer Typst's default spacing then you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
+If you prefer Typst's default spacing, you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
```
Comment (talal) [inline template/main.typ]: I believe, feel free to correct me if I'm mistaken, this change is not due to a grammatical error but rather subjective.

I like how the sentence reads with `then`. Kindly revert this change.

File: template/main.typ [hunk 4]
```diff
@@ -126,7 +126,7 @@ For comparison, here is what `code` in Fira Mono looks like:
 and here is how the same `code` looks in Iosevka:
 #snip("Code snippet typeset in Iosevka font")
 
-In the case that both code snippets look the same then it means that Iosevka is not installed on your computer.
+In case both code snippets look the same, Iosevka is likely not installed on your computer.
```
Comment (talal) [inline template/main.typ]: I prefer the original version. Kindly revert this change.

**Review by talal — CHANGES_REQUESTED** [review]
Thank you for your PR.

I have left some comments regarding the typos.

As for the formatting: when you use this template, a local copy of `main.typ` file is created in your project on the Typst app or on your computer if you use the Typst CLI. If you prefer shorter lines then you can format your local version of `main.typ` with typstfmt.

Kindly revert the formatting.

**Comment by couscousdude** [comment]
Hi, I've reverted the formatting as well as the subjective changes requested.

**Comment by talal** [comment]
Thank you!
